osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/services/EventNotificationMessageService.java

```diff
@@ -88,7 +94,7 @@ public void handleEvent(final String deviceIdentification, final EventNotificati
     }
 
     public void handleEvents(final String deviceIdentification, final List<EventNotificationDto> eventNotifications)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: org.opensmartgridplatform.domain.core.exceptions.UnknownEntityException, org.opensmartgridplatform.shared.exceptionhandling.NotSupportedException [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1160)